### ISSUE TITLE
AVT 2 (webkit) - React Single line and Multi line Code Snippet is cut off and can not scroll to see content

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3735]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
and FireFox Quantum 60.8.0esr (64-bit)
Carbon v10 - React

## Detailed Description
Open React Single line or Multi line Code Snippet and zoom to 200%
(see [WCAG 2.0 S.C. 1.4.4 Resize Text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html))
The scroll bar is present but it does not allow scrolling to see the full code snippet or access to the copy button. (see screenshot)
<img width="1335" alt="Screen Shot 2019-08-14 at 4 32 42 PM" src="https://user-images.githubusercontent.com/21676914/63058510-810bc880-beb2-11e9-926d-b1d26db8fd93.png">
Resize the screen and the scrollbar disappears. Content in the code snippet and the copy button are not visible. (see screenshot)
<img width="533" alt="Screen Shot 2019-08-14 at 4 32 10 PM" src="https://user-images.githubusercontent.com/21676914/63058530-8c5ef400-beb2-11e9-9228-46bbd08e70e0.png">

**Comment by dakahn** [comment]
related #3261

**Comment by abbeyhrt** [comment]
Related conversation from Google Chrome, around the scroll content not being tabbable: https://bugs.chromium.org/p/chromium/issues/detail?id=585413  https://chromium-review.googlesource.com/c/chromium/src/+/1258331 https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/Fku6784p0wI/3MyOMTk7BwAJ

**Comment by asudoh** [comment]
Good finding @abbeyhrt - The GH issue the "intent to ship" refers to (Issue#2851 in https://github.com/whatwg/html repo) shows that it's one in the series of discussions where the spec guys are trying to better specify the area (sequentially/programmatically, etc. focusable).

**Comment by joshblack** [comment]
Hey @snidersd! Could you offer additional steps to reproduce? Couldn't see the behavior you noted

![demo](https://user-images.githubusercontent.com/3901764/70071799-7db06000-15bb-11ea-914b-39a9d54a69fa.gif)

**Comment by snidersd** [comment]
@joshblack Looks like it was fixed somewhere along the way. I can't reproduce the issue any longer.

**Comment by joshblack** [comment]
Awesome! Sounds good. Will close then 😄
